unit-tests/shared/src/test/scala/javalib/math/BigIntegerTest.scala

```diff
@@ -175,4 +176,149 @@ class BigIntegerTest {
 
     assertFalse(jbi1 == jbi2)
   }
+
+  // ported from Scala.js commit cbf86bb dated Oct 23 2020
```

Comment:
@ekrich you mean this comment?

Reply:
I didn't see that down there. I guess we had always put them on the top based on the latest port. I had to put some comments in some of the code I ported as there was some Scala Native specific code so it wasn't a straight port. Added for JDK 9+ might be helpful there.

Reply:
I added it to make it clear where the Scala.js ported test came from.

> Added for JDK 9+ might be helpful there.

I wrote those tests myself. Scala.js does not support these APIs and thus has no tests for them.

Reply:
Sounds good, thanks. It seems sometime it should be cut and dry but is messier in practice.